### PR TITLE
Password exec command

### PR DESCRIPTION
**Pull request by aelgn**
Generate a password externally by calling a shell command.

This will make working in pgAdmin with RDS IAM authentication viable, the passexec cmd will be executed on connect/reconnect if expired as configured under the Advanced tab in the Server dialog.

https://aws.amazon.com/premiumsupport/knowledge-center/rds-postgresql-connect-using-iam/

Example command
`aws --profile MyProfile rds generate-db-auth-token --hostname myhost.rds.amazonaws.com --port 5432 --region my-region --username my_user`

### CODE REVIEW

File: web/pgadmin/browser/server_groups/servers/static/js/server.ui.js [hunk 1]
```diff
@@ -424,7 +426,21 @@ export default class ServerSchema extends BaseUISchema {
...
+        id: 'passexec_cmd', label: gettext('Password exec command'), type: 'text',
+        group: gettext('Advanced'),
+        mode: ['properties', 'edit', 'create'],
+      },
+      {
+        id: 'passexec_expiration', label: gettext('Password exec expiration (seconds)'), type: 'int',
+        group: gettext('Advanced'),
+        mode: ['properties', 'edit', 'create'],
+        visible: function(state) {
+          return !_.isUndefined(state.passexec_cmd) && !_.isNull(state.passexec_cmd);
```
Comment (adityatoshniwal) [inline web/pgadmin/browser/server_groups/servers/static/js/server.ui.js:440]: simply use `!_.isEmpty(state.passexec_cmd)`

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]
@aelgn Nice work there. Just a minor review mentioned below.
Also, please remove the changes for translation files as they're automatically generated before release.
Please update the screenshots as well - https://www.pgadmin.org/docs/pgadmin4/latest/server_dialog.html

**Comment by aelgn** [comment]
> @aelgn Nice work there. Just a minor review mentioned below. Also, please remove the changes for translation files as they're automatically generated before release. Please update the screenshots as well - https://www.pgadmin.org/docs/pgadmin4/latest/server_dialog.html

Thanks. I've updated the PR as requested 👍

**Comment by akshay-joshi** [comment]
Hi @aelgn 

Python style is failing can you please fix those?

**Comment by aelgn** [comment]
@akshay-joshi sure, lint is removed. Not so sure that I agree it looks better, but hey - I'm not here to change the rules ;)

**Pull request merged**